lib/common/options.c

```diff
@@ -246,13 +246,11 @@ static const pcmk__cluster_option_t cluster_options[] = {
             "potentially leading to data loss and/or service unavailability."),
     },
     {
-        // @COMPAT PCMK__ACTION_POWEROFF is disallowed by schema
         PCMK_OPT_STONITH_ACTION, NULL, PCMK_VALUE_SELECT,
-            PCMK_ACTION_REBOOT ", " PCMK_ACTION_OFF ", " PCMK__ACTION_POWEROFF,
+            PCMK_ACTION_REBOOT ", " PCMK_ACTION_OFF,
         PCMK_ACTION_REBOOT, pcmk__is_fencing_action,
         pcmk__opt_schedulerd,
-        N_("Action to send to fence device when a node needs to be fenced "
-            "(\"poweroff\" is a deprecated alias for \"off\")"),
+        N_("Action to send to fence device when a node needs to be fenced"),
```

Comment:
ditto doc/sphinx/Pacemaker_Explained/cluster-options.rst

Reply:
Already done in #3671